src/test/java/com/openmrs/migrator/MigratorApplicationTests.java

```diff
@@ -1,12 +1,14 @@
 package com.openmrs.migrator;
 
+import org.junit.Ignore;
 import org.junit.Test;
 import org.junit.runner.RunWith;
 import org.springframework.boot.test.context.SpringBootTest;
 import org.springframework.test.context.junit4.SpringRunner;
 
 @RunWith(SpringRunner.class)
 @SpringBootTest
+@Ignore
 public class MigratorApplicationTests {
```

Comment:
Lets add one integration test to ensure the run and setup commands run correctly without errors

Reply:
Under tests, create two folders `unit` and `integration` to distinguish between the types of tests

Reply:
Im not sure if that would work, the setup command crates folder/files at  the same level tha app is running

Reply:
but the integration test  was made

Reply:
why would that be a problem in how we structure the test classes?

Reply:
alternatively we could of course just use a naming convention for the types of tests - so something like: `MigratorApplicationIntegrationTests`

Reply:
generally it is a good idea to use different test classes to separate Integration and Unit tests. How we want to structure them doesn't matter too much. We used different folders in the report module, but using naming conventions is good too

Reply:
Hum ok you mean the classes... my first understanding was  split the  test resources  in unit  and integration....

Reply:
As your suggestion  says,  mocking  should be a better idea for  integration test  instead of writing  effectively. What do you think?

Reply:
No so I'm not talking about mocking, there is no need for mocking in our tests atm since we have been able to include PDI with the app. 

What I am referring to is keeping integration tests (like what you have now implemented for MigratorApplication class) separate from unit tests. Take a look at the report module to see what I mean.

Side note: Mocking is for unit tests, an integration test needs to explicitly interface with the actual dependencies to test that all the connections are working as expected. It is too easy to make a mistake and miss something with a mock.

Reply:
Thank you for the explanation (Mocking  usage). 
Now the test classes are separated into unit tests and integration tests